### ISSUE TITLE
AdderNet Upsample

### ISSUE THREAD

**Issue by jdsjfi** [huawei-noah/AdderNet#56]
Hello, I want to realize the Transposed Adder layer according to the relationship between convolution and transposed convolution, but from the result I found that the image it generates has very obvious checkerboard artifacts. I think that it is because the variance of the output of AdderNet is much larger than CNN. So I want to know if you could give me some suggestions to solve it? Thanks very much!!!

**Comment by HantingChen** [comment]
We add BN layer after adder layer to solve this problem in our paper.

**Comment by jdsjfi** [comment]
I also add BN layer after each Transposed Adder layer what I mentioned above and the feature maps it generated also has obvious checkerboard artifacts. Could you help me solve it? Thanks a lot!

**Comment by HantingChen** [comment]
How if you use the transposed conv layer?

**Comment by jdsjfi** [comment]
When I use transposed convolution, it also have a little checkerboard artifacts, but it is not very obvious if you do not observe it carefully.

**Comment by HantingChen** [comment]
I think this may caused by some optimization problems. Maybe you can try to adjust the hyper-parameters such as learning rate.
